### PR TITLE
docs: minor improvements

### PR DESCRIPTION
**Pull request by EazyAl**
Signed-off-by: EazyAl <ali.imran@owkin.com>

### CODE REVIEW

File: substrafl_examples/get_started/plot_substrafl_torch_fedavg.py [hunk 1]
```diff
@@ -210,9 +209,6 @@ def accuracy(datasamples, predictions_path):
 # Specify the machine learning components
 # ***************************************
 #
-# This section uses the PyTorch based SubstraFL API to simplify the definition of machine learning components.
-# However, SubstraFL is compatible with any machine learning framework.
-#
```
Comment (EazyAl) [inline substrafl_examples/get_started/plot_substrafl_torch_fedavg.py:215]: @ThibaultFy I know you suggested keeping this but it is mentioned in detail at the start of this example so putting it again in the middle seems like overkill

**Review by oleobal — APPROVED** [review]
Seems good to me, feel free to wait for feedback from Thibault or Romain before merging if you'd rather.

File: substrafl_examples/get_started/plot_substrafl_torch_fedavg.py (hunk 1 above)
Comment (RomainGoussault) [reply substrafl_examples/get_started/plot_substrafl_torch_fedavg.py:215]: I would keep it, even if it's mentioned at the start. I think it's worth saying it again as it is a super important for user to understand it won't bet the same interface with other FL frameworks.

Comment (ThibaultFy) [reply substrafl_examples/get_started/plot_substrafl_torch_fedavg.py:215]: Pedagogy is the art of repetition 😄

**Review by RomainGoussault — APPROVED** [review]
LGTM, Thanks for the PR!

**Pull request merged**